Makefile

```diff
@@ -268,6 +271,10 @@ verify-previous-version: ## Verifies that PREVIOUS_VERSION variable is set
 bundle-reset-date: ## Reset bundle's createdAt
 	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ${BUNDLE_CSV}
 
+.PHONY: bundle-community-rh
+bundle-community-rh: bundle-community ## Generate bundle manifests and metadata customized to Red Hat community release
+	echo -e "\n  # Annotations for OCP\n  com.redhat.openshift.versions: \"v${OCP_VERSION}\"" >> bundle/metadata/annotations.yaml
+
```

Comment:
Run `$(MAKE) bundle-update` at the end as in `bundle-community` target?

Reply:
Why do it twice?

Reply:
You are right. I haven't noticed that `bundle-community-rh` already calls `bundle-community`...